mkdocs/docs/user_guide/benchmarks.md

```diff
@@ -0,0 +1,102 @@
+ 
+    - Random crop preserving the original aspect ratio (new size - 560 x 420)
+    - Horizontal flip
+    - Vertical flip
+    - 25 degree rotation
+    - Resizing with change in aspect ratio (new aspect ratio - 1:1)
+    
+    Thus, each group has a total of 6 images (original +  transformed). A total of 1800 such groups
+    were created totalling 10800 images in the dataset.
+
+3. Exact duplicate dataset: An image from each of the 2550 image groups of the UKBench dataset was
+ taken and an exact duplicate was created. The number of images totalled 5100.
+ 
+## Metrics
+The metrics used here are classification metrics as explained in the 
+[documentation](https://idealo.github.io/imagededup/user_guide/evaluating_performance/).
+
+class-0 refers to non-duplicate image pairs.
+class-1 refers to duplicate image pairs.
+
+
+## Timings
+The times are reported in seconds and comprise the time taken to generate encodings and find duplicates.
+The time taken to perform the evaluation task is *NOT* reported.
+
+## Results
+### Near Duplicate dataset
+| *Method* | *Threshold* | *Time (s)* | *class-0 precision* |*class-1 precision*  |  *class-0 recall*| *class-1 recall*|
+|--------|-----------|----------|-------------------|-------------------|----------------|----------------|
+| dhash  | 0         | 35.57    | 0.99970585        | 0                 | 1              | 0              |
+| dhash  | 10        | 35.81    | 0.99971922        | 0.01867083        | 0.9992864      | 0.04614379     |
+| dhash  | 32        | 106.67   | 0.998             | 0.00038           | 0.32576433     | 0.88359477     |
+| phash  | 0         | 40.073   | 0.99970589        | 1                 | 1              | 0.00013        |
+| phash  | 10        | 39.056   | 0.9997105         | 0.49896           | 0.99999533     | 0.01581699     |
+| phash  | 32        | 98.835   | 0.998             | 0.0008            | 0.3434116      | 0.85588        |
+| ahash  | 0         | 36.171   | 0.999706          | 0.2828            | 0.9999         | 0.0018         |
+| ahash  | 10        | 36.56    | 0.9997614         | 0.0127            | 0.9956         | 0.1926         |
+| ahash  | 32        | 97.17    | 0.999             | 0.0004            | 0.447443       | 0.93163        |
+| whash  | 0         | 51.71    | 0.99970           | 0.1117            | 0.9999         | 0.0025         |
+| whash  | 10        | 51.94    | 0.99976           | 0.00868           | 0.99334        | 0.1981         |
+| whash  | 32        | 112.56   | 0.999             | 0.0004            | 0.41623        | 0.93300        |
+| cnn    | 0.5       | 379.68   | 0.999999          | 0.002             | 0.8566         | 0.9995         |
+| cnn    | 0.9       | 377.157  | 0.99974           | 0.99488           | 0.9999         | 0.1271         |
+| cnn    | 1.0       | 379.57   | 0.99970           | 0.                | 1.0            | 0.0            |
```

Comment:
I think you wanted to write `0.0` instead of `0.`.

Reply:
Yes, will fix.